src/main/java/commands/SearchCommand.java

```diff
@@ -126,29 +287,129 @@ public void searchExercise() throws InvalidCommandException {
             }
         }
         if (isZeroMatch()) {
-            System.out.println("Sorry, no matching exercise found.");
+            printNotFoundMessage(CATEGORY_EXERCISE);
         }
+        clearMatchCount();
+    }
+
+    /**
+     * Prints message when no matching result is found.
+     *
+     * @param category The category that will be searched.
+     */
+    public void printNotFoundMessage(String category) {
+        System.out.println("Sorry, no matching " + category + " found.");
     }
```

Comment:
This method too, is it possible to bring it to the UI class?

Reply:
This is easy to implement